src/payment/checkout/payment-form/CardDetails.jsx

```diff
@@ -22,6 +23,24 @@ export class CardDetailsComponent extends React.Component {
     };
   }
 
+  componentDidMount() {
+    if (this.props.isPaymentVisualExperiment) {
+      const cardFormats = [
+        '4999 9{4} 9{4} 9{4}', // Visa
```

Comment:
For `inputmask`, `9` represents any number and `{4}` represents 4 of the same thing. So this line corresponds to:
`4xxx xxxx xxxx xxxx` where `x` is any number